xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -64,10 +41,20 @@ struct DemuxPacket;
 #define XBMC_INVALID_CODEC_ID   0
 #define XBMC_INVALID_CODEC      { XBMC_CODEC_TYPE_UNKNOWN, XBMC_INVALID_CODEC_ID }
 
+/*!
+ * @brief Amount of stream properties given via "C" between Kodi and Addon.
+ *
+ * This to know his max allowed size by list with @ref PVR_NAMED_VALUE.
+ *
+ * Used within @ref KodiToAddonFuncTable_PVR by:
+ * - GetEPGTagStreamProperties
+ * - GetChannelStreamProperties
+ * - GetRecordingStreamProperties
+ */
+#define PVR_STREAM_MAX_PROPERTIES 30
+
 /* defines for GetChannelStreamProperties, GetRecordingStreamProperties and GetEPGTagStreamProperties */
-#define PVR_STREAM_MAX_PROPERTIES     20
 #define PVR_STREAM_PROPERTY_STREAMURL "streamurl" /*!< @brief the URL of the stream that should be played. */
-#define PVR_STREAM_PROPERTY_INPUTSTREAMADDON  "inputstreamaddon" /*!< @brief the name of the inputstream add-on that should be used by Kodi to play the stream denoted by PVR_STREAM_PROPERTY_STREAMURL. Leave blank to use Kodi's built-in playing capabilities. */
 #define PVR_STREAM_PROPERTY_INPUTSTREAMCLASS  "inputstreamclass" /*!< @brief the name of the inputstream add-on that should be used by Kodi to play the stream denoted by PVR_STREAM_PROPERTY_STREAMURL. Leave blank to use Kodi's built-in playing capabilities or to allow ffmpeg to handle directly set to `inputstream.ffmpeg`. */
```

Comment:
Just a reminder to add:
```
#define PVR_STREAM_PROPERTY_INPUTSTREAM  "inputstream" /*!< @brief the name of the inputstream that should be used by Kodi to play the stream denoted by PVR_STREAM_PROPERTY_STREAMURL. Leave blank to use Kodi's built-in playing capabilities or to allow ffmpeg to handle directly set to `inputstream.ffmpeg`. */

Reply:
Thanks, I look to change.

Reply:
Thanks, I look to change.

Reply:
You can also delete PVR_STREAM_PROPERTY_INPUTSTREAMCLASS